src/main/java/bio/terra/service/load/LoadDao.java

```diff
@@ -196,6 +199,26 @@ public int getBatchSize() {
         });
   }
 
+  /**
+   * Chunk a stream of models into groups of 1000 and populate the database
```

Comment:
Comment says `1000` but code says `batchSize`
```suggestion
   * Chunk a stream of models into groups of size batchSize and populate the database
```